.storybook/metamask-storybook-theme.js

```diff
@@ -7,4 +7,8 @@ export default create({
   base: 'light',
   brandTitle: 'MetaMask Storybook',
   brandImage: logo,
+
+  // Typography
+  fontBase: 'Euclid, Roboto, Helvetica, Arial, sans-serif',
+  fontCode: 'Inconsolata, monospace',
 });
```

Comment:
Using [storybook theme api](https://storybook.js.org/docs/react/configure/theming) instead of overwriting styles in preview head.